test/scenarios/istio/istio_test.go

```diff
@@ -52,5 +56,18 @@ func TestIstio(t *testing.T) {
 		codemodules.WithProxyAndAGCert(t, proxy.ProxySpec),
 		codemodules.WithProxyCAAndAGCert(t, proxy.HttpsProxySpec),
 	}
-	testEnv.Test(t, feats...)
+
+	filteredFeats := []features.Feature{}
+
+	if cfg.FeatureRegex() != nil {
```

Comment:
Can we extract this part as it is used several times and seems like it is the same (if I am not mistaken)?

Reply:
on it